deps/obs-scripting/obslua/CMakeLists.txt

```diff
@@ -25,26 +25,28 @@ swig_add_library(
 
 add_library(OBS::lua ALIAS obslua)
 
-target_compile_options(obslua PRIVATE "$<$<C_COMPILER_ID:AppleClang,Clang>:-Wno-error=shorten-64-to-32>")
+target_compile_options(
+  obslua
+  PRIVATE $<$<PLATFORM_ID:Windows>:/wd4054>
+          $<$<PLATFORM_ID:Windows>:/wd4197>
+          $<$<PLATFORM_ID:Windows>:/wd4244>
+          $<$<PLATFORM_ID:Windows>:/wd4267>
```

Comment:
Yeah my intention was to have each warning be it's own "unit" or line. I also prefer actual list items vs semicolon-separated strings when providing an actual list of items.

Otherwise the generator expression would need to be wrapped in double quotes with a semicolon between each value, which worsens readability in my opinion.

Reply:
Got it. I find this a bit clearer too, so this can stay as-is.